runtime/src/bank.rs

```diff
@@ -1333,7 +1333,12 @@ impl Bank {
                     self.get_account(&stake_pubkey),
                     self.get_account(&delegation.voter_pubkey),
                 ) {
-                    (Some(stake_account), Some(vote_account)) => {
+                    (Some(mut stake_account), Some(vote_account)) => {
```

Comment:
this is least intrusive way. but a hacky way at the same time.

I'll do more commenting